stats/stats_context.proto

```diff
@@ -16,20 +16,26 @@ package google.instrumentation;
 option java_package = "com.google.instrumentation.stats.proto";
 option java_outer_classname = "StatsContextProto";
 
-// Wire format of instrumentation stats (aka Census) tags.
+// StatsContext describes the encoding of stats context information (tags)
+// for passing across RPC's.
 message StatsContext {
-  // Encodes all stats tags (aka Census tags) to propagate over an RPC in a
-  // single byte vector.
+  // Tags are encoded as a single byte sequence. The format is:
+  // [tag_type key_len key_bytes value_len value_bytes]*
   //
-  // The format is:
-  //  num_tags [key_len key_bytes value_len value_bytes]*
-  //
-  // All implementations must honor these specs:
-  //  * The num_tags is represented using varint.
-  //  * The key/value lengths are represented using varint.
-  //  * All key/values must be US-ASCII.
-  //
-  // For details about varint, see:
-  //  https://developers.google.com/protocol-buffers/docs/encoding#varints
+  // Where:
+  //  * tag_type is one byte, and is used to describe the format of value_bytes.
```

Comment:
I think this description needs to be changed slightly - I've added a suggestion in the shared document.